.gitignore

```diff
@@ -1,2 +1,8 @@
 node_modules
-.db
```

Comment:
keep the `.db` in here, it is where PouchDB writes its files into. Unless we decide to use `pouchdb-adapter-memory` only